jni/nnstreamer.mk

```diff
@@ -60,6 +60,10 @@ NNSTREAMER_DECODER_DV_SRCS := \
 NNSTREAMER_DECODER_IL_SRCS := \
     $(NNSTREAMER_EXT_HOME)/tensor_decoder/tensordec-imagelabel.c
 
+# source Android MediaCodec (AMC)
```

Comment:
At this time,  I am not sure that we have to append gstamcsrc*.c in ./jni/nnstreamer.mk. 
Basically. jni/nnstreamer.mk is intended to guide a basic usage for Android JNI interface.

I think that we may add how to write android source code at nnstreamer-example repository. 
* https://github.com/nnsuite/nnstreamer-example/pull/33

Reply:
OK. I'll fix this.